DMTN-107.tex

```diff
@@ -227,54 +229,60 @@ \subsection{Commissioning Data Templates}\label{ssec:potsol_comm}
+%For example, while it might be possible to cover $\sim50\%$ of the WFD area with one image per filter per field in $\sim$6 photometric nights worth of observing, single-image depth is insufficient for templates (\S~\ref{ssec:req_temp} and \ref{ssec:potsol_imgimg}).
+For example, it might be possible to cover $\sim$50\% of the WFD area with $3$ images per field in a single filter in $\sim$3 photometric nights, which should be sufficient to build template images that meet the noise contribution requirement\ossreq{0158}\reqparam{templateNoiseLevelY1} for LSST year 1 (\S~\ref{ssec:req_temp}).
+However, that is a very idealized minimum time estimate, which assumes 1000 visits per night and does not account for weather or dithering, factors which would $\sim$double the time estimate.
+Such a survey dedicated to template-building would not necessarily serve an independent commissioning-related purpose that would make it worth spending commissioning time on -- it remains the purview of the commissioning and operations teams whether LOY1 alert production is best served by building templates or ensuring a fully-commissioned system.
+
+{\bf (2) A Template-Building "Filler" Program During Commissioning}\\
+During commissioning, whenever activities that require images be taken but are agnostic about their location and filter (e.g., testing telescope parameters, or troubleshooting scenarios), and a filter change and/or telescope slew would not cause an unacceptable overhead or risk (at the discretion of the commissioning team), then obtain images that would be beneficial to building templates for LOY1 alert production.
+An observation scheduler program could be written so that this happens automatically whenever the commissioning team chooses this "Template-Building" mode, and the program could optimize achieving, e.g., $>$3 images in good conditions in a single-filter across as wide an area as possible.
+It is feasible that the equivalent of $\sim$6 photometric nights' worth of time out of LSSTCam commissioning might be spent in this mode, and as described above this could lead to fully filled single-band templates for $\gtrsim$50\% of the WFD main survey area.
+
+{\bf The broader question of what it means to "optimize science".} \\
+For the purposes of this Tech Note we have considered "maximizing science" as being equivalent to "maximizing the number of alerts for unique objects".
+Under this assumption, the priority for template-building during commissioning would be to build single-filter templates over as wide an area as possible.
+The optimal filter debatable: it's likely to be $r$ or $i$-band, and the latter might have the advantage of not requiring airmass correction for DCR (Section 5.2.4 of \citeds{LDM-151}).
+Many surveys have done single-band transient searches in the past with much success (e.g., PTF extragalactic focused on $r$-band). 
```

Comment:
It would be good to reference here the SSSC and TVSSC research notes on maximizing science in LOY1 as examples of other definitions of optimizing science.  And maybe note that such input from the community os valuable.

Reply:
Agreed. I ended up adding this to the end of the following paragraph, which describes the types of scientific studies that would be useful to undertake. It says this:
"Since the first version of this Tech Note, several of the Science Collaborations have begun to compile their science-based recommendations for templates and alerts during LOY1; the Transients and Variable Stars SC has two AAS research notes based on time critical and non-time critical science, \citet{2020RNAAS...4...41S} and \cite{2020RNAAS...4...40H}, and the Solar System SC has a white paper describing their plans for high-impact time-domain science in year one\footnote{\url{http://lsst-sssc.github.io/Files/lsst_templates.pdf}}."